pkg/kubeapiserver/authorizer/modes/modes_test.go

```diff
@@ -20,26 +20,29 @@ import "testing"
 
 func TestIsValidAuthorizationMode(t *testing.T) {
 	var tests = []struct {
+		name      string
 		authzMode string
 		expected  bool
 	}{
-		{"", false},
-		{"rBAC", false},        // not supported
-		{"falsy value", false}, // not supported
-		{"RBAC", true},         // supported
-		{"ABAC", true},         // supported
-		{"Webhook", true},      // supported
-		{"AlwaysAllow", true},  // supported
-		{"AlwaysDeny", true},   // supported
+		{"test1", "", false},
```

Comment:
this doesn't make much sense: why not generate the name automatically, combined with the authzMode in the string (which is unique).